substrate/frame/revive/src/benchmarking/call_builder.rs

```diff
@@ -48,6 +48,7 @@ where
 	T: Config + pallet_balances::Config,
 	BalanceOf<T>: Into<U256> + TryFrom<U256>,
 	MomentOf<T>: Into<U256>,
+	T::Hash: frame_support::traits::IsType<H256>,
```

Comment:
wouldn't T::Hash: Into<H256> suffice for all these added constraints?

Reply:
Technically yes. But I think we are splitting hairs here :) I don't see where a runtime would fullfill `T::Hash: Into<H256>` but `T::Hash` isn't actually `H256`.